inference-engine/tests/functional/plugin/shared/include/behavior/infer_request_config.hpp

```diff
@@ -71,8 +73,10 @@ TEST_P(InferConfigTests, withoutExclusiveAsyncRequests) {
         ASSERT_NO_THROW(ie->SetConfig(config, targetDevice));
     }
     // Load CNNNetwork to target plugins
-    auto execNet = ie->LoadNetwork(cnnNet, targetDevice, config);
-    execNet.CreateInferRequest();
+    if (targetDevice.find(CommonTestUtils::DEVICE_AUTO) == std::string::npos) {
+        auto execNet = ie->LoadNetwork(cnnNet, targetDevice, config);
+        execNet.CreateInferRequest();
```

Comment:
@myshevts we must not have any ifs inside beh tests

Reply:
yes, #6113 is fixing this issue.

Reply:
@ilya-lavrenov , yes, I already pointed to that as well:
https://github.com/openvinotoolkit/openvino/pull/5979#discussion_r647231092

so this PR was merged to meet the FF deadline, and associated ticket to fix the tests was created